test/integration/rpc_test.lua

```diff
@@ -169,11 +169,13 @@ function g.test_routing()
         'myrole', 'void', nil,
         {uri = 'localhost:0'}
     )
-    t.assert_covers({
-        ['"localhost:0": ' .. errno.strerror(errno.ECONNREFUSED)] = true,
-        ['"localhost:0": ' .. errno.strerror(errno.ENETUNREACH)] = true,
-    }, {[err.err] = true})
-    t.assert_equals(err.class_name, 'NetboxConnectError')
+    t.assert_items_include(
```

Comment:
I think it should be done in a different way. 

Please don't use nonexistent ports. Simply you could apply this solution - https://github.com/tarantool/cartridge/pull/696/files#diff-3b3a548c9aeec5e1b0b31b02c38cc44aR160 

Or another way it could be two test. One with ":3303" - allowed port and ":0" - nonexistent ports